src/components/DaySchedule.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import DayShow from "./DayShow";
+import { getDayString } from "../lib/component-utils.js";
+import styles from "../styles/DaySchedule.module.css";
+import { showType } from "../lib/types.js";
+
+export default function DaySchedule({ shows , day}){
+  const showList = shows.filter((s) => getDayString(s.time.day) === day)
+  const showListSorted = showList.sort((a, b)=> {return a.time.hour - b.time.hour});
+
+  //this is stupid code: but its the only solution I can come up with...
```

Comment:
I think the fix is really simple! Try adding `flex-wrap: wrap;` to the CSS inline class. Then you shouldn't need multiple different lists.